mappings/net/minecraft/world/VoxelView.mapping

```diff
@@ -1,4 +1,4 @@
-CLASS net/minecraft/class_4538
+CLASS net/minecraft/class_4538 net/minecraft/world/VoxelView
```

Comment:
We don't usually refer to voxels in yarn do we? It makes sense, but sounds a bit off.

Reply:
What name should we use instead?

Reply:
I think should rather leave this unmapped for later.

Reply:
Renamed to `WorldView` as this is like a reader interface (no modification hooks, really)